boards/waveshare/rp2040_zero/doc/index.rst

```diff
@@ -0,0 +1,150 @@
+.. _rp2040_zero:
+
+Waveshare RP2040-Zero
+#####################
+
+Overview
+********
+
+RP2040-Zero, A Low-Cost, High-Performance Pico-Like MCU Board Based On Raspberry Pi Microcontroller RP2040.
+
+Hardware
+********
+- RP2040 microcontroller chip designed by Raspberry Pi in the United Kingdom.
+- Dual-core Arm Cortex M0+ processor, flexible clock running up to 133 MHz.
+- 264KB of SRAM, and 2MB of on-board Flash memory.
+- USB-C connector, keeps it up to date, easier to use.
+- The castellated module allows soldering direct to carrier boards.
+- USB 1.1 with device and host support.
+- Low-power sleep and dormant modes.
+- Drag-and-drop programming using mass storage over USB.
+- 29 × multi-function GPIO pins (20× via edge pinout, others via solder points).
+- 2 × SPI, 2 × I2C, 2 × UART, 4 × 12-bit ADC, 16 × controllable PWM channels.
+- Accurate clock and timer on-chip.
+- Temperature sensor.
+- Accelerated floating-point libraries on-chip.
+- 8 × Programmable I/O (PIO) state machines for custom peripheral support.
+
+
+.. figure:: img/RP2040-Zero-1.jpg
+     :align: center
+     :alt: RP2040-Zero
+
+
+.. figure:: img/RP2040-Zero-M-1.jpg
+     :align: center
+     :alt: RP2040-Zero with pre-soldered pin header
+
+     RP2040-Zero (above) and RP2040-Zero with pre-soldered pin header (below)
+     (Images courtesy of Waveshare)
+
+Supported Features
+==================
+
+The RP2040-Zero board configuration supports the following hardware features:
```

Comment:
Put board name as used with west with \`\` around it and call it board target, not configuration